libcxx/cmake/caches/cxx03-headers.cmake

```diff
@@ -0,0 +1,2 @@
+set(LIBCXX_TEST_PARAMS "std=c++03;test-cxx03-headers=True" CACHE STRING "")
```

Comment:
I would probably name this new param `use-frozen-cxx03-headers`. That tells what the setting does a bit more explicitly.